internal/state/geometry.go

```diff
@@ -0,0 +1,5 @@
+package state
+
+type Geometry struct {
```

Comment:
Just clarifying, this is an entity to represent how big the player/enemy/crystal is?